### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

**Comment by michielbdejong** [comment]
`DEBUG=* node README.md` now leads to:
```
  ilp-plugin-lightning error connecting to lnd { Error: Stream removed
```
Investigating. This is when lnd is running; if lnd is not running the error is 'Connect Failed' instead.

**Comment by michielbdejong** [comment]
After updating src/rpc.proto to the latest version from lnd, I'm running into:
> Error: failed to import '/Users/michiel/gh/interledgerjs/ilp-plugin-lightning/src/google/api/annotations.proto' in '/Users/michiel/gh/interledgerjs/ilp-plugin-lightning/src/rpc.proto': file not found

Found this: https://github.com/lightningnetwork/lnd/blob/master/docs/grpc/javascript.md

**Comment by michielbdejong** [comment]
With those instructions, got to:
```js
var grpc = require('grpc');
var fs = require("fs");

//  Lnd cert is at ~/.lnd/tls.cert on Linux and
//  ~/Library/Application Support/Lnd/tls.cert on Mac
var lndCert = fs.readFileSync("/Users/michiel/Library/Application Support/Lnd/tls.cert");
var credentials = grpc.credentials.createSsl(lndCert);
var lnrpcDescriptor = grpc.load("rpc.proto");
var lnrpc = lnrpcDescriptor.lnrpc;
var lightning = new lnrpc.Lightning('localhost:10009', credentials);

lightning.getInfo({}, (err, info) => {
  console.log('lightning info callback!', err, info)
})
```
which outputs:
> lightning info callback! { Error: expected 1 macaroon, got 0
    at /Users/michiel/gh/lightningnetwork/lnd/lnrpc/node_modules/grpc/src/client.js:554:15 code: 2, metadata: Metadata { _internal_repr: {} } } undefined

**Comment by emschwartz** [comment]
> Error: failed to import '/Users/michiel/gh/interledgerjs/ilp-plugin-lightning/src/google/api/annotations.proto' in '/Users/michiel/gh/interledgerjs/ilp-plugin-lightning/src/rpc.proto': file not found

I think I ran into that before. There was one line at the top I needed to comment out before in order for it to work

**Comment by michielbdejong** [comment]
Yeah, that's also what that post mentioned. Got it working now, thanks!

**Comment by emschwartz** [comment]
Is this ready for review now?

**Comment by michielbdejong** [comment]
Yes, this is only for updating the dependencies. The script in the readme only shows that you can connect a client and server (they will talk BTP to each other on port 9000 and each talk grpc to LND on port 10009).

I'll write a separate PR to improve readme instructions for setting up a channel and doing an actual transfer. If you prefer, we can keep this PR open and then you can review both at once.

File: README.md [hunk 1]
```diff
@@ -14,22 +14,50 @@ npm install --save ilp-plugin-lightning
 ## Usage
 
 This plugin can be used with the [`ilp`](https://github.com/interledgerjs/ilp) client module or the [`ilp-connector`](https://github.com/interledgerjs/ilp-connector).
+See the [Ledger Plugin Interface](https://github.com/interledger/rfcs/blob/master/0004-ledger-plugin-interface/0004-ledger-plugin-interface.md) for documentation on available methods.
+
+A minimal way to test your setup:
+* Install LND as explained on https://github.com/lightningnetwork/lnd/blob/master/docs/INSTALL.md
+* Run `lnd --bitcoin.active --bitcoin.testnet --debuglevel=debug --neutrino.active --neutrino.connect=faucet.lightning.community --no-macaroons`
+* Once LND is running, you can try the following script (edit the `lndTlsCertPath` and `peerPublicKey` variables accordingly):
```
Comment (emschwartz) [inline README.md]: Where do you get these variables from?

File: README.md [hunk 2]
```diff
@@ -14,22 +14,50 @@ npm install --save ilp-plugin-lightning
...
-const plugin = new PluginLightning({
-  lndUri: 'localhost:11009', // lnd rpc URI
-  // Peer Details
-  rpcUri: 'https://peer.example/rpc',
+const ObjStore = require('ilp-plugin-payment-channel-framework/test/helpers/objStore')
+const PluginLightning = require('.')
+const client = new PluginLightning({
+  server: 'btp+ws://:pass@localhost:9000',
+
+  maxBalance: '1000000', // max allowed balance in Satoshis
```
Comment (emschwartz) [inline README.md]: This description could be more helpful. What exactly is the "max allowed balance"?

**Comment by michielbdejong** [comment]
@emschwartz ready for re-review. I tested this with a btc simnet and everything seems to work now.

**Review by emschwartz — APPROVED** [review]

**Pull request merged**